### ISSUE TITLE
train-TER and train-WER don't change

### ISSUE THREAD

**Issue by tranmanhdat** [facebookresearch/wav2letter#720]
I have followed this [guide](https://github.com/facebookresearch/wav2letter/tree/master/tutorials/1-librispeech_clean) to start trainning.
I have started trainning with docker GPU version on my PC ( 4 GPU RTX2080).
it's ok when i use config in [train.cfg](https://github.com/facebookresearch/wav2letter/blob/master/tutorials/1-librispeech_clean/train.cfg)
![Screenshot from 2020-06-28 17-05-11](https://user-images.githubusercontent.com/30494878/85944565-dd0ffc00-b961-11ea-9605-56cb878b72e4.png)
But when i **change batchsize to 128**, it **does not update train-TER and train-WER too**
![Screenshot from 2020-06-28 16-29-49](https://user-images.githubusercontent.com/30494878/85944590-134d7b80-b962-11ea-8958-898e5aed463d.png)
I checked GPU and it woking well
![Screenshot from 2020-06-28 17-11-32](https://user-images.githubusercontent.com/30494878/85944628-74754f00-b962-11ea-87b1-0bd682fa04f5.png)
I have trained with command:
`mpirun -n 4 --allow-run-as-root ./build/Train train --flagsfile=/root/src/tutorial/1-librispeech_clean/train.cfg --logtostderr=1 --minloglevel=0 --enable_distributed true `

**Comment by tlikhomanenko** [comment]
From your logs you can see that in the second one the loss decreasing very-very slowly. You need to adjust the learning rate when changes the batchsize at least. Why do you want to use such huge total batch size (it is known problem that models with very large batch sizes are harder to make to converge, there are a plenty of papers and studies on this)?

**Comment by tranmanhdat** [comment]
thank you, i will try small batch size, my custom dataset is big so i want a large batch size for faster tranning
